tgui/packages/tgui/styles/interfaces/Safe.scss

```diff
@@ -0,0 +1,78 @@
+.Safe
+{
+  &--engraving
+  {
```

Comment:
Absolute pathing is preferred in SCSS for the same reason we use absolute paths in DM code. It's easier to search, will use less indentation and structure will be flatter.

For consistency sake, place `{` on the same line as the selector.

Here's a short guide to BEM: http://getbem.com/naming/

Fix `--`.

Reply:
Fixed.